.github/workflows/maven.yml

```diff
@@ -19,11 +19,9 @@ name: GitHub CI
 
 on:
   push:
-    branches:
-    - master
+    branches-ignore:
+      - dependabot/**
```

Comment:
Do you want to revert the `yml` changes so that the soup of the PR gets merged ;-)

Reply:
back on this finally. I think it's fine to ignore this to avoid multiple builds running of the same branch.

Reply:
Looking at this with a fresh eye, I don't think this change is needed. It conforms to the [docs](https://docs.github.com/en/code-security/supply-chain-security/keeping-your-dependencies-updated-automatically/automating-dependabot-with-github-actions) and actually you would do the opposite what you mentioned.

Currently, the workflow will start on push to `master` + inside of any Pull Request (so not on branch + PR).
With your change, it would start on *any* branch *except* dependabot branches + on all PRs.

You can see it [here](https://github.com/jenkinsci/jira-plugin/actions), I pushed out a new branch and even pushed out a dummy commit and there were no actions started.